server/src/socket/socket.js

```diff
@@ -184,20 +220,62 @@ module.exports = function (io) {
     }
   };
 
-  var createGameWithFriends = function (player, socket) {
+  const fireRegionGame = (player, socket, region) => {
+    let game;
+    if (!regionGamesNeedingPlayers[region]
```

Comment:
I don't really understand this line. If a region doesn't need players OR (the region needs new players AND the length of that regionGamesNeedingPlayers[region] is 0). I would need more clarity on how an array can be filled and still have a length less than or equal to 0.

Reply:
Okay. Thanks for clarifying. Great job.